test/imported/test_indexing.py

```diff
@@ -98,6 +98,9 @@ def test_index(self):
     numpy_testing_assert_equal_helper(reference[2, 2, 2], 27)
     numpy_testing_assert_equal_helper(reference[:], consec((3, 3, 3)))
 
+    # indexing with single element integer tensor
```

Comment:
don't add test directly to this file, it's a copy of torch's test.

add these tests to test_ops close to other fancy getitem tests

Reply:
Got it, will fix